cairo/ethereum/crypto/kzg.cairo

```diff
@@ -348,3 +349,17 @@ func bytes_to_kzg_commitment{
     }
     return (KZGCommitment(b.value), bool(1));
 }
+
+func bytes_to_kzg_proof{
+    range_check_ptr,
+    bitwise_ptr: BitwiseBuiltin*,
+    range_check96_ptr: felt*,
+    add_mod_ptr: ModBuiltin*,
+    mul_mod_ptr: ModBuiltin*,
+}(b: Bytes48) -> (KZGProof, bool) {
```

Comment:
do you absolutely need to return a boolean flag for success instead of returning an Exception type ?

Reply:
If I understand well in Python this is an AssertionError right?

Reply:
Indeed it could be done with an Exception type

Reply:
I would rather have an exception type for consistency with the codebase: we can use `EthereumException` from `cairo/ethereum/exceptions.cairo`, add a new `const AssertionError = 'AssertionError'`, and return an `EthereumException*`

Reply:
so that in pytest we can use the `try... except... with strict_raises(type(error))` pattern and have the `cairo_run` call return only the non-exception type